dialectRT.go

```diff
@@ -141,15 +141,51 @@ func (d *DialectRT) getMsgById(id uint32) (*dialectMessage, bool) {
 	return &msg, ok
 }
 
+// CreateMessageById returns a DynamicMessage created from finding the message in the DialectRT given corresponding to id.
+func (d *DialectRT) CreateMessageById(id uint32) (*DynamicMessage, error) {
+	dm := &DynamicMessage{}
+	var fields map[string]interface{}
+	msg, ok := d.messages[id]
+	if !ok {
+		errorString := fmt.Sprintf("Message with id=%d does not exist in this dialectRT", id)
+		return dm, errors.New(errorString)
+	}
+	dm.t = msg
+	dm.Fields = fields
+	return dm, nil
+}
+
+// CreateMessageByName returns a DynamicMessage created from finding the message in the DialectRT given corresponding to name.
+func (d *DialectRT) CreateMessageByName(name string) (*DynamicMessage, error) {
+	dm := &DynamicMessage{}
+	var fields map[string]interface{}
+	var msg *dialectMessageRT
+	foundMessage := false
+	for _, m := range d.messages {
+		if m.msg.Name == name {
```

Comment:
This should be 'OriginalName' because that's what you're returning in the `GetName()` method, so that's what I'll be looking for.